src/include/storage/index/b_plus_tree_debug.h

```diff
@@ -0,0 +1,275 @@
+//===----------------------------------------------------------------------===//
+//
+//                         BusTub
+//
+// b_plus_tree_debug.h
+//
+// Identification: bustub/src/include/storage/index/b_plus_tree_debug.h
+//
+// Copyright (c) 2024, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <sstream>
+#include <string>
+
+#include "common/exception.h"
+#include "common/logger.h"
+#include "common/rid.h"
+#include "storage/index/b_plus_tree.h"
+#include "storage/page/b_plus_tree_page.h"
+
+namespace bustub {
+
+/*****************************************************************************
+ * UTILITIES AND DEBUG
+ *****************************************************************************/
+
+/*
+ * This method is used for test only
+ * Read data from file and insert one by one
+ */
+INDEX_TEMPLATE_ARGUMENTS
+void BPLUSTREE_TYPE::InsertFromFile(const std::filesystem::path &file_name) {
+  int64_t key;
+  std::ifstream input(file_name);
+  while (input) {
+    input >> key;
+
+    KeyType index_key;
+    index_key.SetFromInteger(key);
+    RID rid(key);
+    Insert(index_key, rid);
+  }
+}
+/*
+ * This method is used for test only
+ * Read data from file and remove one by one
+ */
+INDEX_TEMPLATE_ARGUMENTS
+void BPLUSTREE_TYPE::RemoveFromFile(const std::filesystem::path &file_name) {
+  int64_t key;
+  std::ifstream input(file_name);
+  while (input) {
+    input >> key;
+    KeyType index_key;
+    index_key.SetFromInteger(key);
+    Remove(index_key);
+  }
+}
+
```

Comment:
```suggestion
/**
 * This method is used for debug only, You don't need to modify
 */
```
Do students need to implement this? Otherwise put on the same comment